pkg/kyma/cmd/test/definitions/cmd_test.go

```diff
@@ -0,0 +1,71 @@
+package definitions
+
+import (
+	"reflect"
+	"testing"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/pkg/api/octopus"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_ListTestDefinitionNames(t *testing.T) {
+	testData := []struct {
+		testName         string
+		shouldFail       bool
+		inputDefinitions oct.TestDefinitionList
+		expectedResult   []string
+	}{
+		{
+			testName:   "correct list",
+			shouldFail: false,
+			inputDefinitions: oct.TestDefinitionList{
+				Items: []oct.TestDefinition{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "test1",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "test2",
+						},
+					},
+				},
+			},
+			expectedResult: []string{"test1", "test2"},
+		},
+		{
+			testName:   "incorrect list",
+			shouldFail: true,
+			inputDefinitions: oct.TestDefinitionList{
+				Items: []oct.TestDefinition{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "test1",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "test2",
+						},
+					},
+				},
+			},
+			expectedResult: []string{"test1", "test3"},
+		},
+	}
+
+	for _, tt := range testData {
+		mCli := octopus.NewMockedOctopusRestClient(&tt.inputDefinitions, nil)
+		dNames, err := listTestDefinitionNames(mCli)
+		if !tt.shouldFail {
+			require.Nil(t, err, tt.testName)
+			require.True(t, reflect.DeepEqual(dNames, tt.expectedResult))
```

Comment:
Is `reflect.DeepEqual()` necessary here? I think `require.Equal()` internally does use `reflect.DeepEqual()`